Libraries/LibWeb/CSS/Length.cpp

```diff
@@ -39,6 +40,28 @@ float Length::relative_length_to_px(const LayoutNode& layout_node) const
         return m_value * layout_node.font_size();
     case Type::Rem:
         return m_value * layout_node.document().document_element()->layout_node()->font_size();
+    case Type::Vw:
+        return layout_node.document().frame()->viewport_rect().width() * (m_value / 100);
+    case Type::Vh:
+        return layout_node.document().frame()->viewport_rect().height() * (m_value / 100);
+    case Type::Vmin: {
+        auto viewport = layout_node.document().frame()->viewport_rect();
+
+        if (viewport.width() > viewport.height()) {
+            return viewport.height() * (m_value / 100);
+        } else {
+            return viewport.width() * (m_value / 100);
+        }
```

Comment:
if you want to be less repetitive/verbose this could be

```suggestion
        return min(viewport.width(), viewport.height()) * (m_value / 100);
```

Reply:
Agreed